packages/react-native-web/src/vendor/Animated/createAnimatedComponent.js

```diff
@@ -15,7 +15,7 @@
 const {AnimatedEvent} = require('./AnimatedEvent');
 const AnimatedProps = require('./nodes/AnimatedProps');
 const React = require('react');
-const ViewStylePropTypes = require('../../components/View/ViewStylePropTypes').default;
+const ViewStylePropTypes = require('../../exports/View/ViewStylePropTypes').default;
```

Comment:
Another hint that `ViewStylePropTypes` (or `StylePropTypes`) should be added to the exported API, allowing modules like `Animated` and `Picker` to be built upon a smaller core API.